lib/watirspec/server.rb

```diff
@@ -54,14 +54,14 @@ def run!
 
             status, headers, body = app.response(path, data)
 
-            headers = headers.merge('Server' => 'ruby')
+            headers['Server'] = 'ruby'
             headers = headers.to_a.map { |array| array.join(': ') }
 
             response = ["HTTP/1.1 #{status}"] + headers
             response = response.join("\r\n")
 
             client.puts(response)
-            client.puts("\r\n")
+            client.puts("\r\n\r\n")
```

Comment:
I see that Travis fails, but i believe that is because it uses Chrome 55 and ChromeDriver 2.24 which are not compatbile.